src/style/blocks/card.css

```diff
@@ -0,0 +1,81 @@
+.card {
+  max-width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__photo {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 40px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin: 0 16px 4px;
+}
+
+.card__title-uppercase {
+  text-transform: uppercase;
+}
+
+.card__product-code {
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin: 0 0 16px 16px;
+}
+
+.card__reviews {
+  float: right;
+  font-size: 10px;
+  line-height: 16px;
+  margin-right: 16px;
+}
+
+.card__stars--margin {
+  margin: 0 0 24px 16px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  font-size: 12px;
+  line-height: 18px;
+  margin: 0 16px 16px;
+  color: #616070;
+}
+
+.card__price-bold {
+  color: #060b35;
+  font-size: 16px;
+  font-weight: 700;
+}
+
+.card__buy {
+  display: block;
+  text-decoration: none;
+  width: 166px;
+  height: 40px;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 40px;
+  text-align: center;
+  text-transform: uppercase;
+  color: #fff;
+  background: #00acdc;
+  border-radius: 5px;
+  border: 1px solid transparent;
+  margin: 0 0 16px 16px;
```

Comment:
do not add horizontal margins for every element add paddings to .card
<img width="235" alt="image" src="https://user-images.githubusercontent.com/66749919/198830670-7d0739bd-f2a6-4ffb-91a6-2af89bb13a06.png">
like this
<img width="212" alt="image" src="https://user-images.githubusercontent.com/66749919/198830855-255b9823-1a33-41a3-b80b-6431c024b18c.png">